react/CircularChart/CircularChartProgress.jsx

```diff
@@ -0,0 +1,59 @@
+import React, { forwardRef } from 'react'
+import PropTypes from 'prop-types'
+
+import { makeStyles } from '../styles'
+import MuiCircularProgress from '../CircularProgress'
+
+const useStyles = makeStyles({
+  root: {
+    position: ({ position }) => position ?? undefined
+  },
+  svg: {
+    transform: 'rotate(-117deg)'
```

Comment:
nit: It is not clear where this number comes from and how it is related to the 65 below. Could you compute one based on the other?